.NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs

```diff
@@ -1,55 +1,55 @@
-//------------------------------------------------------------------------------

```

Comment:
Please be careful to not change all line endings in a file if not really necessary. Can you undo this?